spacy/language.py

```diff
@@ -972,7 +972,8 @@ def __call__(
                 raise ValueError(Errors.E003.format(component=type(proc), name=name))
             try:
                 doc = proc(doc, **component_cfg.get(name, {}))
-            except KeyError:
+            except KeyError as e:
```

Comment:
I think we should just just catch all exceptions here and then raise our `E109` from them. So basically, `raise ValueError(...) from e`. 

This will make the traceback more verbose but it will actually show where the problem happened (with traceback) and will show up as something like "This error was the direct cause of the following...".

I'll just go ahead and update this and then merge your PR 🙂